packages/gatsby-theme-docs/src/components/release-notes-filter-dates.js

```diff
@@ -93,7 +92,6 @@ const ReleaseNotesFilterDates = () => {
             type="date"
             id="to-filter-date"
             max={maximumDate}
-            value={toDate}
```

Comment:
@timonrey Removing value attribute and setting the values directly using DOM selector is a code smell, it means that something else is not working properly. I'll try to have a better look later but this last commit seems not to go in the right direction

Reply:
@timonrey I pushed a commit. The sync between the local state and the application state happens onBlur. I think you can start having a look to it once it's build and fine tune it. But try to stay away from direct DOM interactions unless there's no other solution.

HTH.